dkms.in

```diff
@@ -731,7 +731,7 @@ check_version_sanity()
     local lib_tree="$install_tree/$1" res=
     echo $"Running module version sanity check."
     local i=0
-    if [ -n $3 ]; then
+    if [ -n "$3" ]; then
         # Magic split into array syntax saves trivial awk and cut calls.
```

Comment:
Sorry for dropping the ball here - was busy with work/personal stuff.

Can you add some context for this and the other "fix SCXXXX" commits. For example:

```
dkms: quote the third arg of installed_modules()

As mentioned by shellcheck (SC2070 ... rest of message ...) we need to do X. This has gone unnoticed due to REASON
```

Reply:
clarified synopsis (hopefully)